holoviews/plotting/bokeh/stats.py

```diff
@@ -309,7 +309,7 @@ def get_data(self, element, ranges, style):
             factors = list(unique_iterator(factors))
 
         if self.show_legend:
-            legend_prop = 'legend_field' if bokeh_version >= '1.3.5' else 'legend'
+            legend_prop = 'legend_field' if bokeh_version >= LooseVersion('1.3.5') else 'legend'
```

Comment:
Well caught! I thought I had replaced all of these but seems I missed this one :-(